### ISSUE TITLE
Shouldn't the Reset All button be last in the traversal order?

### ISSUE THREAD

**Issue by Nancy-Salpepi** [phetsims/keplers-laws#160]
**Test device**
MacBook Air M1 chip

**Operating System**
13.5.1

**Browser**
Safari 16.6

**Problem description**
For https://github.com/phetsims/qa/issues/979 I noticed that the Reset All button comes before the measuring tape and stopwatch in the traversal order on all screens. This seems odd to me. Shouldn't the Reset All button be the last thing before heading to the nav bar?

**Steps to reproduce**
1. On any screen, check the stopwatch and measuring tape checkboxes
2. Tab through the order 

**Visuals**

https://github.com/phetsims/keplers-laws/assets/87318828/8ec73926-90d4-4de6-ad00-54595a0b4f09

<details>
Troubleshooting information:
!!!!! DO NOT EDIT !!!!!
Name: ‪Kepler's Laws‬
URL: https://phet-dev.colorado.edu/html/keplers-laws/1.1.0-dev.5/phet/keplers-laws_all_phet.html
Version: 1.1.0-dev.5 2023-09-07 23:32:34 UTC
Features missing: applicationcache, applicationcache, touch
Flags: pixelRatioScaling
User Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/16.6 Safari/605.1.15
Language: en-US
Window: 1317x706
Pixel Ratio: 2/1
WebGL: WebGL 1.0
GLSL: WebGL GLSL ES 1.0 (1.0)
Vendor: WebKit (WebKit WebGL)
Vertex: attribs: 16 varying: 30 uniform: 1024
Texture: size: 16384 imageUnits: 16 (vertex: 16, combined: 32)
Max viewport: 16384x16384
OES_texture_float: true
Dependencies JSON: {}
</details>

**Comment by AgustinVallejo** [comment]
Ready for review. Please close if OK @Nancy-Salpepi

**Comment by Nancy-Salpepi** [comment]
You forgot about the measuring tape 🙂. That still needs to be fixed.

**Comment by AgustinVallejo** [comment]
My bad, should be ready now.
